documentation/projects/proposals/additional_search_views/20230424-project_proposal_additional_search_views.md

```diff
@@ -0,0 +1,36 @@
+# 2023-04-24 Project Proposal
+
+## Reviewers
+
+- [ ] @obulat
+- [ ] @krysal
+
+## Project summary
+
+The Openverse team has decided to enhance the browsing experience by
+incorporating new content views for providers, tags, and creators. This feature
+will allow users to share collections detached of individual searches.
+
+## Goals
+
+Search Experience
+
+## Requirements
+
+## Success
+
+The project will be considered successful when each tag, provider, and creator
+has a standalone and shareable page
+
+## Participants and stakeholders
+
+- Design: @panchovm
+- Implementation:
+
+## Infrastructure
+
+## Accessibility
+
+## Marketing
+
```

Comment:
Since this projects adds views where people can browse content without a search query, there is a comms and marketing opportunity to convey how openverse is moving towards more ways of discovering content. What do you think?

Reply:
Agree! It'd be a good oppportunity to promote the project as well.